packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/CreateRulesTests.java

```diff
@@ -257,6 +271,71 @@ public void createRulesWithOneError()
         restClient.assertLastError().containsSummary("Rule name is a mandatory parameter");
     }
 
+    /** Check we can create a rule without description. */
+    @Test (groups = { TestGroup.REST_API, TestGroup.RULES })
+    public void createRuleWithoutDescription()
+    {
+        RestRuleModel ruleModel = createRuleModelWithDefaultName();
+        UserModel admin = dataUser.getAdminUser();
+
+        RestRuleModel rule = restClient.authenticateUser(admin).withCoreAPI().usingNode(ruleFolder).usingDefaultRuleSet()
+            .createSingleRule(ruleModel);
+
+        restClient.assertStatusCodeIs(CREATED);
+        rule.assertThat().field("id").isNotNull()
+            .assertThat().field("name").is(RULE_NAME_DEFAULT)
+            .assertThat().field("description").isNull();
+    }
+
+    /** Check we can create a rule without specifying triggers but with the default "inbound" value. */
+    @Test (groups = { TestGroup.REST_API, TestGroup.RULES })
+    public void createRuleWithoutTriggers()
+    {
+        RestRuleModel ruleModel = createRuleModelWithDefaultName();
+        UserModel admin = dataUser.getAdminUser();
+
+        RestRuleModel rule = restClient.authenticateUser(admin).withCoreAPI().usingNode(ruleFolder).usingDefaultRuleSet()
+            .createSingleRule(ruleModel);
+
+        restClient.assertStatusCodeIs(CREATED);
+        rule.assertThat().field("id").isNotNull()
+            .assertThat().field("name").is(RULE_NAME_DEFAULT)
+            .assertThat().field("triggers").is(List.of("inbound"));
+    }
+
+    /** Check we can create a rule without error script. */
+    @Test (groups = { TestGroup.REST_API, TestGroup.RULES })
+    public void createRuleWithoutErrorScript()
+    {
+        RestRuleModel ruleModel = createRuleModelWithDefaultName();
```

Comment:
These three tests (createRuleWithoutDescription, createRuleWithoutTriggers and createRuleWithoutErrorScript) make the same API call. Maybe we could combine them into a single test called createRuleWithMinimalFields?